src/main/java/org/toradocu/translator/Matcher.java

```diff
@@ -464,7 +464,7 @@ private static String simpleMatch(String predicate) {
     java.util.regex.Matcher inequalityNumber =
         Pattern.compile(
                 verbs
-                    + "(<=|>=|<|>|!=|==|=)? ?(-?([0-9]+(.[0-9]+)?|zero|one|two|three|four|five|six|seven|eight|nine))")
+                    + "(<=|>=|<|>|!=|==|=)? ?(-?([0-9]+(.[0-9]+)?|zero|one|two|three|four\b|five|six\b|seven\b|eight\b|nine\b)(?! ))")
```

Comment:
Why we consider word boundaries (\b) only for some specific words and not all of them?